Formula/s/seven-kingdoms.rb

```diff
@@ -51,7 +51,10 @@ class SevenKingdoms < Formula
   end
 
   def install
-    system "./configure", "--disable-silent-rules", *std_configure_args
+    args = ["--disable-silent-rules"]
+    args += ["--disable-curl", "--disable-enet", "--disable-multiplayer"] unless Hardware::CPU.intel?
```

Comment:
if these got disabled, is the app still useful?

Reply:
All of these are part of the multiplayer feature:
https://github.com/Homebrew/homebrew-core/blob/1eff6d8763a018b3b1ef02d32a61713822fd7afc/Formula/s/seven-kingdoms.rb#L34-L35

Can still play single player

Reply:
I see 👍